packages/c/cpf-utils.json

```
@@ -0,0 +1,43 @@
+{
+  "name": "cpf-utils",
+  "filename": "cpf-utils.min.js",
+  "homepage": "https://cpf-utils.vercel.app/",
+  "description": "Straight forward JavaScript package to handle CPF data (Brazilian personal ID).",
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Julio L. Muller"
+    }
+  ],
+  "keywords": [
+    "cpf",
+    "validar",
+    "validador",
+    "validate",
+    "validation",
+    "formatar",
+    "format",
+    "gerar",
+    "gerador",
+    "generate",
+    "generator",
+    "pt-br",
+    "br"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/juliolmuller/cpf-utils-js"
+  },
+  "autoupdate": {
+    "source": "npm",
+    "target": "cpf-utils",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
```

```suggestion
          "*.js"
```
We don't accept open globstars in configs as it can lead to files being included on the CDN that we don't expect. It looks like there are just a couple of JS files in the dist directory that are needed?